config/environments/production.rb

```diff
@@ -150,8 +142,15 @@
   # Do not dump schema after migrations.
   config.active_record.dump_schema_after_migration = false
 
-  config.action_mailer.default_url_options = {
-    host: Credentials.fetch(:LIVE_URL_HOST)
-  }
-  Rails.application.routes.default_url_options[:host] = Credentials.fetch(:LIVE_URL_HOST)
+  # Only use :id for inspections in production.
+  config.active_record.attributes_for_inspect = [:id]
```

Comment:
added here https://github.com/rails/rails/blob/v8.0.4/railties/lib/rails/generators/rails/app/templates/config/environments/production.rb.tt#L88

discussion on previous PR
https://github.com/hackclub/hcb/pull/10836#discussion_r2188767745

Reply:
@garyhtou verify we can still look at objects in prod rails console